.github/workflows/wordlist-updater_fuzzing_etc_files.yml

```diff
@@ -0,0 +1,29 @@
+name: update etc files
+
+# Controls when the workflow will run
+on:
+  schedule:
+    - cron: '30 20 1,15 * *' # run at 8:30p on 1st and 15th
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  updatejob:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v3
+
+      # Runs a single command using the runners shell
+      - name: update wordlist
+        run: cd .bin/etc-files-list-update/ && ./update.sh
+
+      - name: print diff
+        run: git diff
+
+      # commit and push
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: update autogenerated files
```

Comment:
To have consistent logging between our github actions, this should be `[Github Action] Updated LFI-etc-files-of-all-linux-packages.txt`